src/main/java/org/squiddev/cobalt/LuaState.java

```diff
@@ -42,6 +41,8 @@
  * Global lua state
  */
 public final class LuaState {
+	public long[] instructionHits = new long[38];
```

Comment:
I don't think is used any more. Can we delete this?

Reply:
Yes. I found it helpful for prioritising optimisations to instructions which occurred frequently in the benchmarks but running `PerformanceTest` with CPU profiling is almost as good.